lib/stdlib/test/digraph_utils_SUITE.erl

```diff
@@ -153,6 +153,8 @@ subgraph(Config) when is_list(Config) ->
     {fg,f,g,fgl} = digraph:edge(SG, fg),
     {fg2,f,g,fgl2} = digraph:edge(SG, fg2),
     {_, {_, acyclic}} = lists:keysearch(cyclicity, 1, digraph:info(SG)),
+    digraph:add_edge(SG, f, g),
+    digraph:add_vertex(SG, b),
```

Comment:
This cannot fail because you are specifying the name of the new vertex. You will need to call `digraph:add_vertex(G)` and check the return value. The graph will also have to contain some vertices created with default names.